FinniversKit/Sources/DNA/Color/Color+SwiftUI.swift

```diff
@@ -4,6 +4,8 @@
 
 import SwiftUI
 
+// swiftlint:disable opening_brace
```

Comment:
What is it complaining about?

Reply:
it's about the spacing between the variable definitions and the braces

```swift
var foo: Color                    { Color.red }
var longVarName: Color  { Color.blue }
```

It complains between the extra spacing in `foo`. Which I just used more as a visual style thing in the code